### DISCUSSION TITLE
Davuluri Hemanth Chowdary

### DISCUSSION THREAD

**Discussion by DavuluriHemanthChowdary** [OpenMined/courses#10] [Introduce Yourself]
Hey ! I'm Hemanth Chowdary currently working as an Full-Stack Application developer!

**Comment by yemikifouly** [comment]
Hello Hemanth! Great to have you here! 🙂 Can I ask, out of curiosity, how you came to be interested in Private AI giving your current area?

**Reply by DavuluriHemanthChowdary** [reply]
I always wanted to get into Data Science or ML industry. I did not have opportunities at my place to work as DS intern or ML intern due to covid-19 so i had to change the plan and i applied for full stack and currently working in that field. This series of courses will be very much helpful to me. Here are few articles i wrote on [medium]( https://medium.com/@hemanthdavuluri)

**Reply by yemikifouly** [reply]
Hemanth! Those articles look great. I saw you have a few focused on NLP. Do you have  a specific interest in NLP, you know, as opposed to other areas like CV or RecSys? Also, I am sure that you will find the series very helpful. 🙂 Have you started the first course yet?

**Reply by DavuluriHemanthChowdary** [reply]
I will sure look into those.  As I'm a pretty beginner and I wish I utilize this opportunity. And Yes I started the course and will finish it soon!
